### ISSUE TITLE
Request: Changelog

### ISSUE THREAD

**Issue by ferhatelmas** [Masterminds/squirrel#226]
There aren't regular releases and it's hard to follow changes. 

Is it possible to provide a changelog and release more frequently? Thank you!

**Comment by lann** [comment]
Squirrel has been effectively inactive (or more optimistically, "complete") for a while now. I have updated the README to reflect this fact.

**Comment by ferhatelmas** [comment]
@lann thanks. 

There was an application running with a commit hash from >2 years ago and I wanted to upgrade to the latest version. I looked for a changelog for a sanity check :wink:
